pulsar/dlq_router.go

```diff
@@ -155,7 +156,7 @@ func (r *dlqRouter) getProducer(schema Schema) Producer {
 	}
 
 	// Retry to create producer indefinitely
-	backoff := &internal.DefaultBackoff{}
+	bo := &backoff.DefaultBackoff{}
```

Comment:
Could you pass the backoff to the `dlqRouter`?

Reply:
Done.

Reply:
DefaultBackoff will still be used?